cg/meta/orders/case_submitter.py

```diff
@@ -276,22 +276,22 @@ def _create_link(self, case_obj, family_samples, father_obj, mother_obj, sample)
 
     def _create_sample(self, case, customer_obj, order, ordered, sample, ticket):
```

Comment:
Not in this PR, but should not there be a pydantc sample model instead of a sample dict?

Reply:
Maybe. The whole ordering part of cg already has three or four different sample models, so I think one might want to clean it all up instead of adding one more.

Reply:
Sounds like a very good focused PR that will bring a lot of benefit in the future 👍